.github/workflows/test_coverage.yml

```diff
@@ -0,0 +1,21 @@
+name: Code Coverage
+
+on:
+  push:
+    branches: [ master, unstable ]
+  pull_request:
+    branches: [ master, unstable ]
+
+jobs:
+  test-coverage-upload:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
```

Comment:
Also, add a blank line between the `actions/checkout` step and the next step `Install cargo-tarpaulin`, for better organization (nitpick).